sound/soc/sof/intel/ext_manifest.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ */
+
+/*
+ * Intel extended manifest is a extra place to store Intel cavs specific
+ * metadata about firmware, for example LPRO/HPRO configuration is
+ * Intel cavs specific. This part of output binary is not signed.
+ */
+
+#ifndef __INTEL_CAVS_EXT_MANIFEST_H__
+#define __INTEL_CAVS_EXT_MANIFEST_H__
+
+#include <sound/sof/ext_manifest.h>
+
+/* EXT_MAN_ELEM_PLATFORM_CONFIG_DATA elements identificators */
+enum sof_cavs_config_elem_type {
+	SOF_EXT_MAN_CAVS_CONFIG_EMPTY		= 0,
```

Comment:
CAVS ?  Now I am working on SOF driver + CAVS FW, so I used it in my project, what is your CAVS ? thanks

Reply:
@RanderWang The existing naming has been that "cAVS" is name of the harwdware. For example: https://thesofproject.github.io/latest/platforms/intel-cavs/index.html

Reply:
I guess the name "cAVS" firmware made sense when it was the only one, before SOF existed.

Shameless plug: https://github.com/thesofproject/sof/issues/3491

Reply:
But the actual patch seems still correct. This is a config element type that is specific to firmware running on cAVS hardware (to separate from elements that are common to all hardware supported by SOF).